packages/react/src/components/Chat/ChatMessage.tsx

```diff
@@ -50,6 +50,9 @@ export interface ChatMessageProps
   /** Menu with actions of the message. */
   actionMenu?: ShorthandValue
 
+  /** Controls messages's relation to other chat messages. It is automatically set from the ChatItem */
```

Comment:
nit-nit: lets end description with dot, also, we might consider to use `by` instead of `from`, something like:

```
 Controls messages's relation to other chat messages. Is automatically set by the ChatItem.
```